src/box/errcode.h

```diff
@@ -99,10 +99,12 @@ struct errcode_record {
 	_(ER_TEST_TYPE_STRING, 10008,		"Test error", "field", STRING) \
 	_(ER_TEST_TYPE_MSGPACK, 10009,		"Test error", "field", MSGPACK) \
 	_(ER_TEST_TYPE_TUPLE, 10010,		"Test error", "field", TUPLE) \
-	_(ER_TEST_2_ARGS, 10011,		"Test error", "f1", INT, "f2", INT) \
-	_(ER_TEST_3_ARGS, 10012,		"Test error", "f1", INT, "f2", INT, "f3", INT) \
-	_(ER_TEST_4_ARGS, 10013,		"Test error", "f1", INT, "f2", INT, "f3", INT, "f4", INT) \
-	_(ER_TEST_5_ARGS, 10014,		"Test error", "f1", INT, "f2", INT, "f3", INT, "f4", INT, "f5", INT) \
+	_(ER_TEST_2_ARGS, 10030,		"Test error", "f1", INT, "f2", INT) \
```

Comment:
Nit: Let's make the gaps even bigger then: 10101, 10102, 10103, ..., then 10201, 10202, 10203, ..., 10301, 10302, 10303, ...

Reply:
Done.

<details>
<summary>diff</summary>

```diff
diff --git a/src/box/errcode.h b/src/box/errcode.h
index 25814f89f..dc42482ff 100644
--- a/src/box/errcode.h
+++ b/src/box/errcode.h
@@ -99,20 +99,20 @@ struct errcode_record {
        _(ER_TEST_TYPE_STRING, 10008,           "Test error", "field", STRING) \
        _(ER_TEST_TYPE_MSGPACK, 10009,          "Test error", "field", MSGPACK) \
        _(ER_TEST_TYPE_TUPLE, 10010,            "Test error", "field", TUPLE) \
-       _(ER_TEST_2_ARGS, 10030,                "Test error", "f1", INT, "f2", INT) \
-       _(ER_TEST_3_ARGS, 10031,                "Test error", "f1", INT, "f2", INT, "f3", INT) \
-       _(ER_TEST_4_ARGS, 10032,                "Test error", "f1", INT, "f2", INT, "f3", INT, "f4", INT) \
-       _(ER_TEST_5_ARGS, 10033,                "Test error", "f1", INT, "f2", INT, "f3", INT, "f4", INT, "f5", INT) \
-       _(ER_TEST_FORMAT_MSG, 10050,            "Test error %i %s", "f1", INT, "f2", STRING) \
-       _(ER_TEST_FORMAT_MSG_LESS, 10051,       "Test error %i %s", "f1", INT, "f2", STRING, "f3", INT) \
-       _(ER_TEST_OMIT_TYPE_CHAR, 10060,        "Test error %c", NULL, CHAR) \
-       _(ER_TEST_OMIT_TYPE_INT, 10061,         "Test error %i", NULL, INT) \
-       _(ER_TEST_OMIT_TYPE_UINT, 10062,        "Test error %u", NULL, UINT) \
-       _(ER_TEST_OMIT_TYPE_LONG, 10063,        "Test error %li", NULL, LONG) \
-       _(ER_TEST_OMIT_TYPE_ULONG, 10064,       "Test error %lu", NULL, ULONG) \
-       _(ER_TEST_OMIT_TYPE_LLONG, 10065,       "Test error %lli", NULL, LLONG) \
-       _(ER_TEST_OMIT_TYPE_ULLONG, 10066,      "Test error %llu", NULL, ULLONG) \
-       _(ER_TEST_OMIT_TYPE_STRING, 10067,      "Test error %s", NULL, STRING) \
+       _(ER_TEST_2_ARGS, 10100,                "Test error", "f1", INT, "f2", INT) \
+       _(ER_TEST_3_ARGS, 10101,                "Test error", "f1", INT, "f2", INT, "f3", INT) \
+       _(ER_TEST_4_ARGS, 10102,                "Test error", "f1", INT, "f2", INT, "f3", INT, "f4", INT) \
+       _(ER_TEST_5_ARGS, 10103,                "Test error", "f1", INT, "f2", INT, "f3", INT, "f4", INT, "f5", INT) \
+       _(ER_TEST_FORMAT_MSG, 10200,            "Test error %i %s", "f1", INT, "f2", STRING) \
+       _(ER_TEST_FORMAT_MSG_LESS, 10201,       "Test error %i %s", "f1", INT, "f2", STRING, "f3", INT) \
+       _(ER_TEST_OMIT_TYPE_CHAR, 10300,        "Test error %c", NULL, CHAR) \
+       _(ER_TEST_OMIT_TYPE_INT, 10301,         "Test error %i", NULL, INT) \
+       _(ER_TEST_OMIT_TYPE_UINT, 10302,        "Test error %u", NULL, UINT) \
+       _(ER_TEST_OMIT_TYPE_LONG, 10303,        "Test error %li", NULL, LONG) \
+       _(ER_TEST_OMIT_TYPE_ULONG, 10304,       "Test error %lu", NULL, ULONG) \
+       _(ER_TEST_OMIT_TYPE_LLONG, 10305,       "Test error %lli", NULL, LLONG) \
+       _(ER_TEST_OMIT_TYPE_ULLONG, 10306,      "Test error %llu", NULL, ULLONG) \
+       _(ER_TEST_OMIT_TYPE_STRING, 10307,      "Test error %s", NULL, STRING) \

 #else
 #define TEST_ERROR_CODES(_)
```

</summary>